Packs/OpenSourceVulnerabilities/Integrations/OSV/OSV.yml

```diff
@@ -0,0 +1,214 @@
+      description: ""
+      type: String
+  - arguments:
+    - description: ' The version string to query for. A fuzzy match is done against
+        upstream versions. Eg. 3.3.0'
+      name: version
+      required: true
+    - description: The name of the package/project to query for. Eg. django-tinymce
+      name: packageName
+      required: true
+    - description: The ecosystem of the package. Eg. PyPI
+      name: ecosystem
+    description: Query vulnerabilities for a particular project based on package name
+      and verion
+    name: osv-query-affected-by-package
+    outputs:
+    - contextPath: OSV.VulnerabilityList.vulns.id
+      description: The `id` field is a unique identifier for the vulnerability entry.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.published
+      description: The RFC3339 timestamp indicating when this entry was published.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.modified
+      description: The RFC3339 timestamp indicating when this entry was last modified.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.withdrawn
+      description: Optional. The RFC3339 timestamp indicating when this entry is considered
+        to be withdrawn.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.summary
+      description: Required. One line human readable summary for the vulnerability.
+        It is recommended to keep this under 120 characters.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.details
+      description: Required. Any additional human readable details for the vulnerability.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.affected.ranges.repo
+      description: Required if type is GIT. The publicly accessible URL of the repo
+        that can be directly passed to clone commands.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.affected.ranges.events.introduced
+      description: The earliest version/commit where this vulnerability was introduced
+        in.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.affected.ranges.events.fixed
+      description: The version/commit that this vulnerability was fixed in.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.affected.ranges.events.limit
+      description: The limit to apply to the range.
+      type: string
+    - contextPath: OSV.VulnerabilityList.vulns.references.url
+      description: Required. The URL.
+      type: string
+    - contextPath: OSV.VulnerabilityList.code
+      type: number
+    - contextPath: OSV.VulnerabilityList.message
+      type: string
+    - contextPath: OSV.VulnerabilityList.details.typeUrl
+      type: string
+    - contextPath: OSV.VulnerabilityList.details.value
+      type: string
+  dockerimage: demisto/python3:3.9.9.25564
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
```

Comment:
```suggestion
  type: python
  fromversion: 6.0.0
```

Reply:
Is there a way to do this from XSOAR UI?

Reply:
@anilagr Unfortunately there isn't, but you can do it from this page by clicking on the `Commit suggestion` button.